### ISSUE TITLE
First slide 1px height -> Safari on first load

### ISSUE THREAD

**Issue by tatof** [kenwheeler/slick#3395]
**> short description of the bug / issue, provide more detail below.**
When the slide is loaded for the first time in safare the slide is only 1px height.
Chrome "had" the same issue but I fixed that with: 
.slick-slide { height: auto; }
==================================================================

http://jsfiddle.net/fmo50w7n/2817/
===================================================================

#### Steps to reproduce the problem
1. Copied the same stuff from site to jsfiddle
2. done

Got the same problem on 6 different sites alle with the latest jquery and slack

**Comment by Mecaurio** [comment]
I have this same problem in Chrome too

**Comment by tatof** [comment]
Yes we are not the only one.. if you search you find plenty topics but without a stabel good solution :(
